cmd/mdatagen/loader.go

```diff
@@ -158,12 +158,22 @@ type attribute struct {
 	Description string `validate:"notblank"`
 	// NameOverride can be used to override the attribute name.
 	NameOverride string `mapstructure:"name_override"`
+	// Enabled defines whether the attribute is enabled by default.
+	Enabled *bool `yaml:"enabled" validate:"required"`
```

Comment:
Why pointer?

Reply:
This was actually needed during the transition from the old metrics builder. Now it can be a bool value, not a pointer. I'll submit a PR for metric.Enabled

Reply:
I removed the pointer from metrics in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16775. @hughesjj you probably will need to rebase